.github/workflows/check.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
-      - run: cargo check --tests --examples
+      - run: cargo check --tests --examples    
```

Comment:
Trailing white spaces... Don't hate me :)

Reply:
Missed that, thank you!